app/migrations/Version20160310181749.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace Sylius\Migrations;
```

Comment:
I think there should be license block here - @pjedrzejewski did we agreed about this?